framework/axSGreaseSkeleton.js

```diff
@@ -11,6 +11,57 @@
 
 /*** Functions for common tweaks. ***/
 
+/**
+ * Adds text to the given live region, and clears it a second later so it's no
+ * longer perceivable.
+ * @param {string} regionid an id of a region.
+ */
+function announce(text, regionId) {
+	getLiveRegion(regionId)
+		.then((region) => {
+			region.innerText = text;
+			setTimeout(() => {
+				region.innerText = '';
+			}, 1000);
+		});
+}
+
+/**
+ * create or fetch a live region that can be used with updateLiveRegion. Returns a promise with the region.
+ * @param {string} id the name of the new live region. This is an html id.
+ * @return {!Promise<HTMLElement>} a div that contains the live region. This can typically be ignored, this exxists to aid in chaining creation of non-existant regions.
```

Comment:
```suggestion
 * @return {!Promise<HTMLElement>} a div that contains the live region. This can typically be ignored, this exists to aid in chaining creation of non-existant regions.
```